satellitevu/apis/archive.py

```diff
@@ -17,6 +22,41 @@ def search(
         page_token: Optional[str] = None,
         **kwargs,
     ):
+        """
+        Perform a search on Satellite Vu's STAC. Relevant documentation is located
+        at https://api.satellitevu.com/archive/v1/docs#operation/Search_search_post
+
+        Args:
+            intersects: Optional dictionary with keys "coordinates" and "geometry"
+            type that search results intersect with. Available geometry types include:
+            "Point","MultiPoint", "LineString", "MultiLineString", "Polygon",
+            "MultiPolygon". For example:
+            intersects = {"coordinates":[-1.065151, 51.163899], "type" : "Point"}
+
+            date_from: Optional datetime representing the start date of the search
+
+            date_to: Optional datetime representing the end date of the search
+
+            limit:  Number of search results (defaults to 25) to be returned.
+
+        Kwargs:
+            bbox: List of points (min longitude, min latitude, max longitude,
+            max latitude) specifying a bounding box to search within.
+
+            ids: List of strings specifying the desired STAC identifiers
+            e.g. ["20221010T222611000_basic_0_TABI"]
+
+            collections: List of strings specifying the collections within which
+            to search. Values include "basic" and "relative".
+
+            sortby: List of fields specifying the field and direction the results
+            are sorted by e.g. [{"field": "datetime", "direction": "desc"}]
+
+        Returns:
+            A FeatureCollection (dictionary containing keys: "type", "features",
```

Comment:
It's returning an `ResponseWrapper` instance, where the `.json()` method returns the FeatureCollection.